### ISSUE TITLE
Prisma crashed when trying to create a new migration

### ISSUE THREAD

**Issue by ElayadeIsmail** [prisma/prisma#14490]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`migrate dev --name [redacted]`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v16.13.2           | 
| Prisma CLI  | 4.1.0              | 
| Engine      | 8d8414deb360336e4698a65aa45a1fbaf1ce13d8| 

## Error
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs:908:22] We should only be setting a changed default if there was one on the previous schema and in the next with the same enum.

Please create an issue with your `schema.prisma` at
https://github.com/prisma/prisma/issues/new

```

**Comment by jkomyno** [comment]
Also reported [here](https://prisma-errors.netlify.app/report/14168).

**Comment by Jolg42** [comment]
Hi @ElayadeIsmail 👋🏼 
It looks like this was fixed in 4.7.0, could you try upgrading to that version or a more recent one?

PR: https://github.com/prisma/prisma-engines/pull/3379

If this is not already fixed, we would need more information here to get to a reproduction.

**Comment by Jolg42** [comment]
Closing this, if you still experience this on a recent version of Prisma, please create a new issue.
